src/warplib/generateWarplib.ts

```diff
@@ -22,40 +9,14 @@ import { glob } from 'glob';
 import { parseMultipleRawCairoFunctions } from '../utils/cairoParsing';
 
 const warplibFunctions: WarplibFunctionInfo[] = [
-  add(),
-  add_unsafe(),
-  add_signed(),
-  add_signed_unsafe(),
   // sub - handwritten
-  sub_unsafe(),
-  sub_signed(),
-  sub_signed_unsafe(),
-  mul(),
-  mul_unsafe(),
-  mul_signed(),
-  mul_signed_unsafe(),
   // div - handwritten
   // div_unsafe - handwritten
-  div_signed(),
-  div_signed_unsafe(),
   // mod - handwritten
-  mod_signed(),
-  exp(),
-  exp_signed(),
-  exp_unsafe(),
-  exp_signed_unsafe(),
-  negate(),
-  shl(),
-  shr(),
-  shr_signed(),
   // ge - handwritten
-  ge_signed(),
   // gt - handwritten
-  gt_signed(),
   // le - handwritten
-  le_signed(),
   // lt - handwritten
-  lt_signed(),
   // and - handwritten
   // xor - handwritten
```

Comment:
Comments should be updated as well since the functions are not handwritten anymore